### PR TITLE
[INLONG-6063][Manager] Utilize ClickHouse and Iceberg data node

### LINKED ISSUE #6063: [Feature][Manager] Utilize ClickHouse and Iceberg data node
### Description

Utilize ClickHouse and Iceberg data node

### Use case

clickhouse or iceberg as sink

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by woofyzhao**
- Fixes #6063

Modifications:
- read from data node  when ck url is not supplied
- read from iceberg data node when iceberg catalog uri is not supplied
- remove jdbc url from hive data node because it's the same as url in base class.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/iceberg/IcebergSinkOperator.java [hunk 1]
```diff
@@ -83,6 +87,15 @@ public StreamSink getFromEntity(StreamSinkEntity entity) {
         }
 
         IcebergSinkDTO dto = IcebergSinkDTO.getFromJson(entity.getExtParams());
+        if (StringUtils.isBlank(dto.getCatalogUri()) && dto.getCatalogType().equals(CATALOG_TYPE_HIVE)) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/iceberg/IcebergSinkOperator.java:90]: Suggest changing `dto.getCatalogType().equals(CATALOG_TYPE_HIVE)` to `CATALOG_TYPE_HIVE.equals(dto.getCatalogType())`.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/iceberg/IcebergResourceOperator.java [hunk 2]
```diff
@@ -78,11 +87,27 @@ public void createSinkResource(SinkInfo sinkInfo) {
         this.createTable(sinkInfo);
     }
 
+    private IcebergSinkDTO getIcebergInfo(SinkInfo sinkInfo) {
+        IcebergSinkDTO icebergInfo = IcebergSinkDTO.getFromJson(sinkInfo.getExtParams());
+
+        // read uri from data node if not supplied by user
+        if (StringUtils.isBlank(icebergInfo.getCatalogUri())
+                && icebergInfo.getCatalogType().equals(CATALOG_TYPE_HIVE)) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/iceberg/IcebergResourceOperator.java:95]: ditto.

**Review by healchow — APPROVED** [review]
LGTM

**Review by yunqingmoswu — APPROVED** [review]
LGTM

**Comment by gong** [comment]
I suggest add description about modifying hive

**Review by dockerzhang — APPROVED** [review]

**Comment by woofyzhao** [comment]
> I suggest add description about modifying hive

Done. See the third  modification desc above.

**Pull request merged**